docs/vercel/src/Index.svelte

```diff
@@ -82,4 +125,32 @@
     text-align: center;
     padding: 0.5rem 0;
   }
+
+  .title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 3rem;
+    margin-bottom: 1.5rem;
+  }
+
+  .title h3 {
+    margin-bottom: 0;
+    margin-top: 0;
+  }
+
+  .checkbox-group {
+    margin-top: 1.5rem;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+  }
+
+  .checkbox-group label {
+    text-transform: capitalize;
+  }
+
+  .show {
+    display: none;
+  }
```

Comment:
The logic is a bit reversed here :slightly_smiling_face:

Perhaps rename this class to `.hide`?